### PR TITLE
Update PHP support

### PR DESCRIPTION
**Pull request by henriquemoody**
Composer will not install this package when the PHP version is 8.0. That
is because it requires either ~5.5 or ^7.0.

Also, support to PHP 5 is not wanted anymore.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     }
   },
   "require": {
-    "php": "~5.5|^7.0",
+    "php": ">=5.5",
```
Comment (malukenho) [inline composer.json]: Although it __works__ out of the box, as PHP doesn't follow semantic versioning we cannot rely on it.

**Edit**: we shouldn't really maintain php `5.x` versions anymore. We can change it allow version `7` and `8` if that is the case.

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ php:
   - 7.1
```
Comment (malukenho) [inline .travis.yml:6]: Can we remove [non-maintained php versions](https://www.php.net/supported-versions.php) on that session? We can add other newer versions as well.

Comment (henriquemoody) [reply .travis.yml:6]: I also added 7.3

**Comment by malukenho** [comment]
Thank you @henriquemoody 🍾 
I still have some work to do before tagging a new version, I hope I can do it until tomorrow.

**Pull request merged**